superset-frontend/src/common/components/index.tsx

```diff
@@ -43,6 +43,11 @@ export const MenuItem = styled(AntdMenu.Item)`
   > a {
     text-decoration: none;
   }
+
+  &.ant-menu-item {
+    height: 30px;
+    line-height: 30px;
+  }
```

Comment:
Are there any Menu.Items that do _not_ have the `.ant-menu-item` class?

If it's all the same, it might be cleaner to do this block like so:

```suggestion
  height: 30px;
  line-height: 30px;
  
  > a {
    text-decoration: none;
  }
```

Also, it might make sense for the height/line-height to be based on typography, i.e. if the font-size is `theme.typography.sizes.m` (14px) then maybe these `30px` values should be `theme.typography.sizes.m * 2`

Reply:
I _think_ I did that to increase specificity... I'll check again if that works without `&.ant-menu-item`.
Good point on using `theme.typography.sizes.m * 2`, will change

Reply:
This same issue appeared on another PR: https://github.com/apache/incubator-superset/pull/11487

One way or another, we should fix it there too :)

Reply:
Done.
Also, I verified that `&.ant-menu-item` syntax is necessary - if we don't use it, antd's styles have higher specificity.

Reply:
There's another trick that might work:
```
&& {
 ...styles
}
```
... which would double-select the emotion classname to bump the specificity point value. Anyway, that's weird too, so no worries. At least we're not using `!important`!